CODE/Sources/PennyBOT/Handlers/InteractionHandler.swift

```diff
@@ -277,6 +296,46 @@ private extension InteractionHandler {
                     }
                 }
             }
+        case let .help(helpMode):
+            switch helpMode {
+            case .add:
+                let name = try modal.components
+                    .requireComponent(customId: "name")
+                    .requireTextInput()
+                    .value.requireValue()
+                    .trimmingCharacters(in: .whitespaces)
+                    .replacingOccurrences(of: "\n", with: " ")
```

Comment:
Do we actually want to permit help-text names with newlines in them, or is this a matter of Discord being too liberal with line breaks?

Reply:
Discord allows new lines, so here I'm just replacing those new lines with a simple space incase someone actually sent a multi-line name.

Reply:
I'd be of a mind to just reject such names as invalid, honestly.

Reply:
hmmm ... ok